src/stylesheets/base/_vars.scss

```diff
@@ -1,2 +1,34 @@
+/*--------- Font Faces ------*/
+
+@font-face {
+    font-family: geomanist;
+    src: url(fonts/geomanist-regular-webfont.woff2);
+}
+
+@font-face {
+    font-family: geomanist-medium;
+    src: url(fonts/geomanist-medium-webfont.woff2);
+}
+
+$main-font: geomanist;
+$main-font-medium: geomanist-medium;
+$secondary-font: Merriweather;
+$tertiary-font: 'Myriad Pro';
+
+/*-------- Colors ------*/
+
 $florence-blue: #184d66;
+$florence-light-blue: #d2e7e2;
 $florence-orange: #c06d3a;
+$florence-blue-green: #0e5254;
+$florence-red: #e95e67;
+$white: #ffffff;
+$florence-gray: #504949;
+
+/*------ Breakpoints ----*/
+
+$screen-xs: 400px;
+$screen-sm: 768px;
+$screen-md: 1000px;
+$screen-lg: 1200px;
+$screen-xl: 1400px;
```

Comment:
You could even include your min/max widths in your screen variables here, e.g.:
```
$screen-sm: 'max-width: 768px'
```
Then there would be less to type when doing media queries in your css, e.g. 
```
@media ($screen-sm) {...}
```